assembl/static2/js/app/pages/tokenVote.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
```

Comment:
A mon avis, cette page n'est pas nécessaire. On pourrait simplement faire un cas spécifique dans la barre de navigation de la V2. Si l'`identifier` de la phase courante est `token` (il y a une méthode dans `utils/timeline` pour récupérer l'`identifier` de la phase courante), on fait un `window.location` lorsqu'on clique sur le bouton débat de la bare de navigation.

Reply:
Je suis pas fan de cette solution, parce que ça fait écrire du code de redirection au mauvais endroit selon moi et que plusieurs types de liens à plusieurs endroits peuvent emmener vers la page de vote ("débat" en haut, "je participe" juste en dessous dans la landing page, l'image de phase dans la landing page, et le lien dans le sous-menu de navigation listant les phases).

J'ai essayé de coder la redirection directement dans `routes.jsx` mais sans succès, ça semble être une limitation de `react-router`. On en a discuté avec @ayyazdaniaryan et on a essayé d'implémenter autre chose à base de `<Redirect>` sans succès non plus. Puis il a commencé à chercher d'autres solutions et va bientôt implémenter quelque chose, notamment https://stackoverflow.com/questions/42914666/react-router-external-link/44563899#44563899 qui ressemble beaucoup à la solution que j'ai utilisée.

Reply:
Oui effectivement, j'ai pas poussé la reflection assez loin. Il y a plusieurs bouton qui permette de rejoindre le débat...

Reply:
I've made the necessary changes here: 2c9070345ed5ea87a91821660953e5cf88b26e1f 

@swergas You can update all links pointing to the token page in react by using the `oldVote` route and it should work without a problem. No URL flashing, quick redirection, and functional.